gerrit_changes_to_github.py

```diff
@@ -271,6 +279,83 @@ def is_repository_usable(args):
     return got_gerrit and got_target
 
 
+def get_events_from_gerrit(gerrit_password, ghpa_token):
+    """On success a list of event-dicts are returned. On error, None is returned."""
+
+    event_url = "https://review.spdk.io/a/plugins/events-log/events/"
+
+    try:
+        last_timestamp = int(os.getenv("LAST_TIMESTAMP"))
+    except Exception as e:
+        log.error(f"Error reading LAST_TIMESTAMP env. var.; err({e})")
+        return
+
+    since = datetime.fromtimestamp(last_timestamp, tz=timezone.utc).strftime(
+        "%Y-%m-%d %H:%M:%S"
+    )
+    log.info(f"Using last timestamp (UTC): {since}")
+
+    # Set t2 cutoff to avoid getting a fraction of events for a particular second
+    last_timestamp = int(datetime.now().timestamp()) - 1
+    until = datetime.fromtimestamp(last_timestamp, tz=timezone.utc).strftime(
+        "%Y-%m-%d %H:%M:%S"
+    )
+
+    if (
+        response := requests.get(
+            f"{event_url}/?t1={since};t2={until}",
+            auth=("spdk-community-ci-samsung", gerrit_password),
```

Comment:
Would it make sense to move this username and the other URLs out of the code and into a cli-argument?